shared/domains/chelonia/time-sync.js

```diff
@@ -0,0 +1,95 @@
+import sbp from '@sbp/sbp'
+
+let wallBase = Date.now()
+let monotonicBase = performance.now()
```

Comment:
Could you add comments here explaining what `wallBase` and `monotonicBase` mean? These terms might be confusing to people (as they are confusing to me)